src/api/data/cache/charthelper/chart_package_helper.go

```diff
@@ -134,6 +136,19 @@ var ensureChartDataDir = func(chart *models.ChartPackage) error {
 	return nil
 }
 
+var cleanChartDataDir = func(chart *models.ChartPackage) error {
+	dir := chartDataDir(chart)
+	if _, err := os.Stat(dir); err != nil {
```

Comment:
Fair point, I was copying the same structure for `ensureDataDir` but I think you're right in that this could be much simpler.